ci/ci/ci.py

```diff
@@ -395,6 +396,18 @@ async def batch_callback_handler(request):
             for wb in watched_branches:
                 if wb.branch.short_str() == target_branch:
                     log.info(f'watched_branch {wb.branch.short_str()} notify batch changed')
+
+                    if 'test' in params and params['complete']:
+                        assert 'deploy' not in params
+                        assert 'dev' not in params
+                        db: Database = app['db']
```

Comment:
Is your thinking here that if the batch callback is missed, then the cleanup loop will eventually cleanup the database once the namespace expires in a day?

Reply:
Yup!